engine/objconv/asteroidgen.cpp

```diff
@@ -144,9 +144,9 @@ class Tri
 struct asteroid
 {
     Vector center;
-    float  radius;
+    float  radius = 0; //Initializing to a value so the compiler shuts up about it
```

Comment:
should use a float (`0.0f`) to initialize a float; compiler will cast it any way so not a big deal.

Reply:
Done!